IDE/iotsafe/user_settings.h

```diff
@@ -0,0 +1,113 @@
+/* user-settings.h
+ *
+ * Copyright (C) 2006-2020 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL. (formerly known as CyaSSL)
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+/* Example 'user-settings.h' for IoT-Safe demo */
+#include <stdint.h>
+
+#define WOLFSSL_IOTSAFE
+#define HAVE_IOTSAFE_HWRNG
+#define HAVE_HASHDRBG
+
+#define WOLFSSL_GENERAL_ALIGNMENT 4
+#define DEBUG_WOLFSSL
+#define WOLFSSL_LOG_PRINTF
+#define DEBUG_WOLFSSL_VERBOSE
+#define SINGLE_THREADED
+#define WOLFSSL_USER_IO
+
+#define TIME_OVERRIDES
+
+extern volatile unsigned long jiffies;
+static inline long XTIME(long *x) { return jiffies;}
+#define NO_ASN_TIME
+#define WOLFSSL_USER_CURRTIME
+#define NO_OLD_RNGNAME
+#define SMALL_SESSION_CACHE
+#define WOLFSSL_SMALL_STACK
+#define TFM_ARM
+#define TFM_TIMING_RESISTANT
+#define ECC_TIMING_RESISTANT
+
+
+/* Connect IoT-safe with PK_CALLBACKS */
+#define HAVE_PK_CALLBACKS
+
+/* ECC definitions */
+#   define HAVE_ECC
+#   define FP_ECC
```

Comment:
Do you really need fixed point caching enabled (also FP_LUT) for this example?

Reply:
leftover, removed.